packages/plugin-graphql/src/schema/schema.js

```diff
@@ -1,50 +1,78 @@
 const { makeExecutableSchema } = require('apollo-server-express');
 const { configTypeDefs, configResolvers } = require('./config');
 const { graphTypeDefs, graphResolvers } = require('./graph');
+const fs = require('fs');
 const gql = require('graphql-tag');
+const path = require('path');
 
-module.exports = (graph) => {
-  let uniqueCustomDataDefKeys = {};
-  let customDataDefs = '';
+module.exports = (compilation) => {
+  const { graph } = compilation;
+  const uniqueCustomDataDefKeys = {};
```

Comment:
Naming / organization could maybe some thought here, just so it's clearer what everything and where it is coming from.